external/loaders/loaders/batches/_batch.py

```diff
@@ -121,6 +122,8 @@ def batches_from_mapper(
     unstacked_dims: Optional[Sequence[str]] = None,
     subsample_ratio: float = 1.0,
     drop_nans: bool = False,
+    shuffle_timesteps: bool = True,
+    shuffle_samples: bool = True,
```

Comment:
If you're changing the numbers output anyways by going from shuffle-select_first to select_fraction, I'd suggest setting shuffle_samples to False by default since this is what we expect to want to use in essentially every case.

As noted in other comment, shuffle_timesteps can be True always and removed as an option.